erpnext/public/js/controllers/transaction.js

```diff
@@ -1911,7 +1911,7 @@ erpnext.TransactionController = erpnext.taxes_and_totals.extend({
 	set_query_for_item_tax_template: function(doc, cdt, cdn) {
 		var item = frappe.get_doc(cdt, cdn);
 		if(!item.item_code) {
-			frappe.throw(__("Please enter Item Code to get item taxes"));
+			// Do nothing
 		} else {
```

Comment:
@casesolved-co-uk Apologies, the use case was not as clear to me. This seems very valid, i checked if it breaks anything. Server side validations are in place, so all good.

**A change I'd suggest:**
Instead of `if-else`, have just `if(item.item_code)` and the lines below in it.
We don't need a snippet to do nothing.

Reply:
Now done